### ISSUE TITLE
Links in logtail.html assume it's the index page for this directory

### ISSUE THREAD

**Issue by danielrichman** [ukhas/js-logtail#10]
https://github.com/ukhas/js-logtail/blob/master/logtail.html#L13 should perhaps be `?` and `?noreverse`.

**Comment by danielrichman** [comment]
Reported by @oppybob

**Comment by oppybob** [comment]
Hi Daniel,
It works perfectly now.
How can I make it work in a more general case ? (I’m willing to learn, I worked years ago with html and asp (before ajax))
Eventually point me at what to read.
Thank you
Bob

**Comment by danielrichman** [comment]
Do you mean to say that you made the change I suggested, and it worked?

**Comment by oppybob** [comment]
yes but I would like to make it work for a page that is not the index

**Comment by danielrichman** [comment]
I think that changing these lines like so

``` html
            <a href="?">Reversed</a> or
            <a href="?noreverse">chronological</a> view.
```

might do it, but I can't test it now.

**Comment by oppybob** [comment]
perfect
thank you
